### ISSUE TITLE
Does this code get 68.01 map?

### ISSUE THREAD

**Issue by auroua** [DetectionTeamUCAS/RRPN_Faster_RCNN_Tensorflow#1]

**Comment by yangxue0827** [comment]
Not this project, the 68+ project is an improved version of the current open source repo and will not be open source recently. @auroua

**Comment by auroua** [comment]
What's the `map` of this released code?

**Comment by yangxue0827** [comment]
I will update the readme after I get the exact result.

**Comment by auroua** [comment]
OK, Thanks~~

**Comment by GaoShanwen** [comment]
Hello, excuse me. I noticed that the parameters of your paper named R2CNN++ are set as follows:
* overlap:200
* subimages size:800 × 800
* iterations:20k
* weight decay :0.0001
* momentum:0.9
* mini-batch size:512
* anchor ratios:[1/1,1/2,1/3,1/4,1/5,1/6,1/7,1/9]
* base anchor size to 256
* anchor scales setting from 2−4 to 2
* MomentumOptimizer
* data augmentation:only random flip(training) 
* different R-NMS thresholds

So what are the parameters of the RRPN project when the map is 61.01, and which is the backbone, Faster or FPN?And the convbody is R-50 or R-101?

 Looking forward to your reply.Thank you!

**Comment by yangxue0827** [comment]
https://github.com/DetectionTeamUCAS/RRPN_Faster-RCNN_Tensorflow/blob/master/libs/configs/cfgs.py @GaoShanwen

**Comment by GaoShanwen** [comment]
Oh,,,I know，Thank you for your reply!
